tests/test_fp8_gemm_onednn.py

```diff
@@ -3,59 +3,150 @@
 import torch
 
 from tests.ops.fp8_quant_op import scaled_fp8_quant
-from tests.register_ops import fp8_gemm_w8a16
+from tests.register_ops import fp8_gemm, fp8_gemm_w8a16
 
 BATCHES = [1]
 MNK_FACTORS = [
-    (1, 1, 1),
     (1, 4, 8),
-    (1, 7, 5),
-    (13, 17, 67),
-    (26, 37, 13),
-    (67, 13, 11),
-    (257, 13, 11),
-    (658, 13, 11),
+    (2, 4, 8),
+    (4, 32, 16),
+    (16, 32, 32),
```

Comment:
can we add miniscope for simulator test? also add some real shape used in popular model?